keyboards/xd75/keymaps/buzzlighter1/layers.json

```diff
@@ -0,0 +1 @@
+[["KC_GESC", "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_6", "KC_7", "KC_8", "KC_9", "KC_0", "KC_MINS", "KC_EQL", "KC_DEL", "KC_BSPC", "KC_TAB", "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P", "KC_LBRC", "KC_RBRC", "KC_BSLS", "KC_PAUS", "KC_CAPS", "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K", "KC_L", "KC_SCLN", "KC_QUOT", "KC_NO", "KC_ENT", "KC_NO", "KC_LSFT", "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "KC_SLSH", "KC_RSFT", "MO(1)", "KC_UP", "KC_NO", "KC_LCTL", "MO(0)", "KC_LALT", "KC_NO", "KC_SPC", "KC_NO", "KC_HOME", "KC_END", "KC_PGUP", "KC_PGDN", "TG(1)", "KC_RCTL", "KC_LEFT", "KC_DOWN", "KC_RGHT"], ["KC_NO", "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_F6", "KC_F7", "KC_F8", "KC_F9", "KC_F10", "KC_F11", "KC_F12", "KC_NO", "KC_NO", "KC_NO", "RGB_TOG", "RGB_MOD", "RGB_RMOD", "RGB_VAI", "RGB_VAD", "KC_P7", "KC_P8", "KC_P9", "KC_PMNS", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "BL_TOGG", "BL_INC", "BL_DEC", "BL_STEP", "KC_NO", "KC_P4", "KC_P5", "KC_P6", "KC_PPLS", "KC_NLCK", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_P1", "KC_P2", "KC_P3", "KC_PSLS", "KC_PENT", "KC_NO", "KC_TRNS", "KC_NO", "KC_NO", "KC_NO", "KC_TRNS", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_P0", "KC_PDOT", "KC_PAST", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO"]]
```

Comment:
This file should be deleted IMO. It serves no purpose in this state.

If you use the command line, you can run `git rm --cached -- keyboards/xd75/keymaps/buzzlighter1/layers.json` to remove this file from version control, but keep it on your system.